packages/nimble-storybook/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "@ni-private/nimble-storybook",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "description": "Storybook for the Nimble repo",
+  "scripts": {
+    "build": "tsc -p ./tsconfig.json && storybook build -o dist/storybook --webpack-stats-json",
+    "start": "storybook dev -p 6006",
+    "lint": "eslint .",
+    "format": "eslint . --fix"
+  },
+  "dependencies": {
+    "@microsoft/fast-react-wrapper": "^0.3.22",
+    "@ni/nimble-components": "^26.0.0",
+    "@ni/spright-components": "^0.0.2",
+    "@ni/nimble-tokens": "^6.13.2",
```

Private packages that are not published can depend on monorepo packages with * to avoid unnecessary package file updates

```suggestion
    "@ni/nimble-components": "*",
    "@ni/spright-components": "*",
    "@ni/nimble-tokens": "*",
```